src/main/resources/db/migration/v0.38.0.29__agm_agm_association_loads_2.sql

```diff
@@ -0,0 +1,30 @@
+ALTER TABLE agmagmassociation
+    add column
+        agmAgmAssociationObject_id bigint
+;
+
+alter table agmagmassociation
+    DROP agmassociationobject_id
+;
+
+delete
+from bulkscheduledload
+where id in (SELECT id
+             FROM bulkload
+             WHERE backendbulkloadtype = 'AGM_AGM_ASSOCIATION');
+
+delete
+from bulkmanualload
+where id in (SELECT id
+             FROM bulkload
+             WHERE backendbulkloadtype = 'AGM_AGM_ASSOCIATION');
+
+delete
+from bulkload
+WHERE group_id in (select id from bulkloadgroup where name = 'Direct (LinkML) AGM AGM Association Loads');
+
+delete
+from bulkloadgroup
+where name = 'Direct (LinkML) AGM AGM Association Loads';
+
+
```

Comment:
The field names, indexes and FK's need to be consistent. 

Alpha has: 
Columns: agmassociationsubject_id, agmassociationobject_id
Indexes: agmagmassociation_agmassocobject_in, agmagmassociation_agmassocsubject_in
FKs: agmstrassociation_agmassociationobject_fk, agmstrassociation_agmassocsubject_fk

I would assume when this change is done that we should have:

Columns: agmagmassociationSubject_id, agmagmassociationobject_id
Indexes: agmagmassociation_agmassocsubject_index, agmagmassociation_agmassocobject_index
FKs: agmagmassociation_agmassociationsubject_fk, agmagmassociation_agmassociationobject_fk